### PR TITLE
Dual Stack OVN-Kubernetes job

### PR DESCRIPTION
**Pull request by aojea**
**- What this PR does and why is it needed**

Set up a periodic job to test dual stack

**- Special notes for reviewers**

This does not affect merging, since the job added runes periodically and is only informative

**- How to verify it**

You can see an example of how it runs here
https://github.com/aojea/ovn-kubernetes/actions/runs/184296553

**- Description for the changelog**

### CODE REVIEW

**Comment by aojea** [comment]
Results here, it is working 
https://github.com/aojea/ovn-kubernetes/runs/899312656

**Comment by aojea** [comment]
/assign @dcbw 

> Results here, it is working
https://github.com/aojea/ovn-kubernetes/runs/899312656

**Comment by danwinship** [comment]
yeah lgtm too. I don't really know the CI setup stuff though

**Comment by Billy99** [comment]
So this just runs periodic job on master? Why would we limit it to just dual-stack? For the IPv6 CI PR, I limited the test matrix just to reduce the test time for each PR. The periodic run could test those additional matrix items in the background for sanity.

**Comment by aojea** [comment]
> So this just runs periodic job on master? Why would we limit it to just dual-stack? For the IPv6 CI PR, I limited the test matrix just to reduce the test time for each PR. The periodic run could test those additional matrix items in the background for sanity.

1 step at a time :smile: 

we can add more periodic jobs later, we just need dual stack ASAP

also, @Billy99 , if we merge this https://github.com/ovn-org/ovn-kubernetes/pull/1500, we can enable more combinations on presubmit

File: .github/workflows/test_periodic.yml [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+        popd
+
+    - uses: actions/download-artifact@v2
+      with:
+        name: test-image
+    - name: Load docker image
+      run: |
+        docker load --input image.tar
+
+    - name: kind setup
```
Comment (Billy99) [inline .github/workflows/test_periodic.yml:170]: Will this be replaced with `install-kind.sh` at some time in the future?  Assume you need additional patches to KIND for dual-stack (https://github.com/aojea/kind/releases/download/dualstack/kind).

File: .github/workflows/test_periodic.yml [hunk 2]
```diff
@@ -0,0 +1,226 @@
+name: ovn-ci-periodic
```
Comment (Billy99) [inline .github/workflows/test_periodic.yml:1]: Will this file have to always be separate from test.yml? Always get nervous when there is a lot the same or real similar logic in two places and trying to make sure they stay in synch.

File: .github/workflows/test_periodic.yml (hunk 1 above)
Comment (aojea) [reply .github/workflows/test_periodic.yml:170]: yeah, but right now is a chicken an egg problem, we need to stabilize dual-stack first.
This way we avoid any risk of affecting other things in the repo, and allows us to have signal on master

File: .github/workflows/test_periodic.yml (hunk 2 above)
Comment (aojea) [reply .github/workflows/test_periodic.yml:1]: same comment as in here https://github.com/ovn-org/ovn-kubernetes/pull/1520#discussion_r461001228 

until it is stable, I expect lot of churn in this file, we can do any changes here without impacting the rest of the jobs

**Comment by dcbw** [comment]
@aojea I'll merge becuase we have a bunch of LGTMs, but can you answer Billy's questions too? Thanks!

**Pull request merged**